content/tutorials/single-page-tutorials/wallet-setup.md

```diff
@@ -0,0 +1,431 @@
+
+If you created a new mnemonic seed phrase at the beginning of this tutorial, you should see an entirely empty balance on your account.
+
+```json
+<Account Name>'s Balance:
+{
+  baseCoin: { total: '0', available: '0' },
+  requiredStorageDeposit: '0',
+  nativeTokens: [],
+  nfts: [],
+  aliases: [],
+  foundries: [],
+  potentiallyLockedOutputs: {}
+}
+```
+
+***
+
+## Request tokens from faucet
+
+Below you can find two ways to request funds from the Shimmer testnet faucet. You can either request funds via the Shimmer faucet website, or programmatically by directly calling the API of the faucet.
+
+### Request tokens via faucet website
+
+Now you can go to the [Shimmer Testnet Faucet Website](https://faucet.testnet.shimmer.network/), paste in your newly created address and request funds.
+
+:::note
+
+The 'Request' button will become clickable as soon as a valid Shimmer address is recognized.
+
+:::
+
+![Shimmer Testnet Faucet](./images/shimmer-testnet-faucet.png)
+
+***
+
+### Request tokens via faucet API
+
+Create a new file `request-faucet.js` and add the code below:
+
+Make sure to paste your previously generated address into the variable `receivingAddress`. The function `requestFunds` receives the URL of the faucet API as well as a valid Shimmer testnet address and sends a request to the faucet API for tokens.
+
+```javascript
+// Libraries
+const fetch = require("node-fetch");
+
+// Network configuration
+const { networkConfig } = require("./networkConfig.js");
+const faucetApi = networkConfig.faucetApi;
+
+// Address to receive faucet tokens
+const receivingAddress = '<paste_your_previously_generated_address_here>';
+
+
+async function run() {
+    const request = await requestFunds(faucetApi, receivingAddress);
+    console.log(request);
+}
+
+// Request tokens from faucet via API call
+async function requestFunds(faucetUrl, addressBech32) {
+    const requestFunds = await fetch(faucetUrl, {
+            method: 'POST',
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({ address: addressBech32 })
+        });
+
+    return await requestFunds.json();
+}
+
+run();
+```
+
+Run the script `request-faucet.js` and check the console output:
+
+```console
+node request-faucet.js
+```
+
+If the request was successfull, the console output should look similar to this:
+
+```json
+{
+  address: '<your_address>',
+  waitingRequests: 1
+}
+```
+
+***
+
+### Check balance again
+
+After a few seconds you can check your balance again by running the script `check-balance.js`:
+
+```console
+node check-balance.js
+```
+
+If the faucet successfully transferred testnet tokens to your address, your balance should look similar to the content below. In case the total balance for the `baseCoin` is still 0, repeat the process of running the `check-balance.js` script, since it might take a little until the funds have been transferred:

```

Comment:
```suggestion
If the faucet successfully transferred testnet tokens to your address, your balance should look similar to the content below. If the total balance for the `baseCoin` is still 0, repeat the process of running the `check-balance.js` script since it might take a little time for the funds to be transferred.
```